spec/metaractor_spec.rb

```diff
@@ -256,13 +260,27 @@ def call
         result = error_test_class.call
         expect(result).to be_failure
 
-        expect(result.errors).to include({
-          source: '/user',
-          title: 'Bad user'
+        expect(result.errors.to_h).to include({
+          base: 'Invalid configuration',
+          is_admin: 'must be true or false',
+          user: {
+            title: 'cannot be blank',
+            username: ['must be unique', 'must not be blank']
+          }
         })
 
         expect(result.error_messages).to include(
-          '/user: Bad user'
+          'Invalid configuration',
+          'is_admin must be true or false',
+          'user.title cannot be blank',
+          'user.username must be unique',
+          'user.username must not be blank'
+        )
+
+        expect(result.errors.full_messages_for(:user)).to include(
```

Comment:
I do like this.

Reply:
Yeah I lifted that from `ActiveModel::Errors`. It's a nice interface.